mne/decoding/csp.py

```diff
@@ -232,7 +232,7 @@ def fit(self, X, y):
         eigen_vectors = eigen_vectors[:, ix]
 
         self.filters_ = eigen_vectors.T
-        self.patterns_ = linalg.pinv(eigen_vectors)
+        self.patterns_ = linalg.pinv2(eigen_vectors)
```

Comment:
why this change?

Reply:
`pinv` failed for me on some problem where `pinv2` succeeded